components/eam/src/chemistry/modal_aero/modal_aero_convproc.F90

```diff
@@ -472,8 +486,7 @@ subroutine ma_convproc_intr( state, ptend, pbuf, ztodt,             &
         else
            l = lmassptr_amode(ll,n)
         end if
-
-        call outfld( trim(cnst_name(l))//'SFWET', aerdepwetis(:,l), pcols, lchnk )
+!        call outfld( trim(cnst_name(l))//'SFWET', aerdepwetis(:,l), pcols, lchnk )
```

Comment:
Unless there is a strong reason for keeping it, this line can be deleted rather than commented out.